frontend/src/components/admin/ManageStoryTranslations.tsx

```diff
@@ -18,6 +23,14 @@ const ManageStoryTranslations = () => {
   const [level, setLevel] = useState<string | null>(null);
   const [stage, setStage] = useState<string | null>(null);
   const [searchText, setSearchText] = useState<string | null>(null);
+  const [numTranslationsInTranslation, setNumTranslationsInTranslation] =
+    useState<number | null>(null);
```

Comment:
nit: I think it would be fine for the default state to be `0` and the state type to be `number`